ts-rs/src/serde_json.rs

```diff
@@ -0,0 +1,48 @@
+use std::{collections::HashMap, path::Path};
+
+use super::{impl_primitives, impl_shadow, typelist::TypeList, TS};
+
+// Manually implement `TS` for `Value`.
+// Defining an untagged enum and deriving `TS` doesn't work for the following reasons:
+// - `#[derive(TS)]` doesn't work in this crate, since the macro generates `::ts_rs::TS`
+// - `Value` is self-referential, and in this case (an untagged enum), tsc doesn't compile
+//   `type JsonValue = ... | Record<string, JsonValue>`, but `{ [x: string]: JsonValue }` works.
+impl TS for serde_json::Value {
+    type WithoutGenerics = Self;
+
```

Comment:
The `impl` is pretty exactly like the one generated by 
```rs
#[derive(TS)]
#[ts(untagged, export_to = "serde_json/")]
enum JsonValue {
    String(String),
    Number(i32),
    Array(Vec<TsJsonValue>),
    Object(HashMap<String, JsonValue>),
}
```
with the exception that it uses `{ [key: string]: JsonValue }` instead of `Record<string, JsonValue>`.